borsh/src/schema.rs

```diff
@@ -63,7 +61,7 @@ pub struct BorshSchemaContainer {
     /// Declaration of the type.
     pub declaration: Declaration,
     /// All definitions needed to deserialize the given type.
-    pub definitions: HashMap<Declaration, Definition>,
+    pub definitions: Vec<(Declaration, Definition)>,
```

Comment:
This changes a public API, this shouldn't be done lightly. See https://github.com/near/borsh-rs/issues/51 for the context on things which *could* be better, but which we can't *just* do, as that'll break the world.